dotcom-rendering/src/web/components/DynamicFast.tsx

```diff
@@ -485,9 +485,9 @@ export const DynamicFast = ({
 		firstSliceLayout = 'oneVeryBig';
 		firstSliceCards = groupedTrails.veryBig.slice(0, 1);
 	} else if (groupedTrails.veryBig.length > 1) {
-		if (groupedTrails.veryBig[0].isBoosted) {
+		if (groupedTrails.veryBig[0]?.isBoosted) {
```

Comment:
This PR was made with `noUncheckedIndexedAccess` enabled, these changes are done in preparation for enabling it across the codebase.